### DISCUSSION TITLE
How to enable CORS in Flutter Web with Canvaskit as renderer?

### DISCUSSION THREAD

**Discussion by vipafattal** [nhost/nhost#2659] [Q&A]
I have already tried whitelisting my Flutter web app in Nhost Cloud. So far it works but when using the canvaskit renderer I get Access to XMLHttpRequest at 'https://SUBDOMAIN.storage.eu-central-1.nhost.run/v1/files/7fdd38aa-2bc9-4625-b55f-405611d9ac32' from origin  has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

BTW I am using Network.Image

**Comment by dbarrosop** [comment]
I just tried `curl -v -X OPTIONS -H "Origin: myapp.com" https://subdomain.storage.eu-central-1.nhost.run/v1/files/7fdd38aa-2bc9-4625-b55f-405611d9ac32` (replacing the URL with one of my own). And I can see the access control headers being returned correctly:

```
< access-control-allow-methods: GET,PUT,POST,HEAD,DELETE
< access-control-max-age: 43200
< accept-ranges: bytes
< access-control-allow-origin: myapp.com
< access-control-expose-headers: Content-Length,Content-Type,Cache-Control,Etag,Last-Modified,X-Error
```

I'd suggest trying yourself and see if you see the headers as well.
